arch.sh

```diff
@@ -96,7 +96,8 @@ generate_new_ramdisk() {
       echo "blacklist i915"
       echo "### mei driver blacklisted due to serious bugs"
       echo "blacklist mei"
-      echo "blacklist mei-me"
+      echo "blacklist mei_me"
+      echo 'blacklist sm750fb'
```

Comment:
this should probably be consistent across all operatingsystems and in a centralized place. Maybe we should keep that in mind for a future cleanup.

Reply:
I don't think all of these modules apply to *all* distributions in *all* versions.
Some OS releases ship buggy kernels/modules and only there we should blacklist.

However the intel `mei` and `mei_me` modules should always be blacklisted IMHO.